styles/landing.css

```diff
@@ -0,0 +1,109 @@
+@import url("./main.css");
+
+.products-nav {
+  display: flex;
+  justify-content: space-around;
+}
+
+.product {
+  font-weight: bold;
+  padding: 10px;
+}
+
+.product:hover {
+  background-color: var(--primary-color);
+  color: var(--secondary-color);
+}
+
+.section-b {
+  background-image: url("https://c4.wallpaperflare.com/wallpaper/984/693/125/wheel-tires-wheels-company-wallpaper-preview.jpg");
+  height: 70vh;
+  background-size: cover;
+  background-position: center;
+  margin-top: 1rem;
+}
+
+.intro {
+  color: var(--secondary-color);
+  padding-top: 2rem;
+  padding-left: 6rem;
+}
+
+.main-title {
+  margin-bottom: 2rem;
+  font-size: 1.8rem;
+  text-transform: uppercase;
+  font-weight: bold;
+}
+
+.usp-points {
+  padding-bottom: 2rem;
+  font-weight: bold;
+}
+
+.bullet-icon {
+  color: #4e9f3d;
+  margin-right: 10px;
+}
+
+.title-brand {
+  text-transform: capitalize;
+}
+.brands {
+  height: 60vh;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+}
+
+.about-brand {
+  width: 55rem;
+}
+
+.brand-names {
```

Comment:
You can create a utility class for it named as .flex-full-center{ //properties here} and use it wherever needed